src/field/gf_101_2.rs

```diff
@@ -52,24 +50,26 @@ impl<F: FiniteField> From<F> for QuadraticPlutoField<F> {
 impl<F: FiniteField> FiniteField for QuadraticPlutoField<F> {
   type Storage = u32;
 
+  const NEG_ONE: Self = Self::new(F::NEG_ONE, F::ZERO);
+  // TODO: This is wrong
+  const ONE: Self = Self::new(F::ONE, F::ZERO);
```

Comment:
What should this be?

Reply:
i think it's a clippy issue, it reorganised the comments somehow, not sure.